server/sonar-qa-util/src/main/java/org/sonarqube/qa/util/pageobjects/UsersManagementItem.java

```diff
@@ -0,0 +1,98 @@
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with this program; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ */
+package org.sonarqube.qa.util.pageobjects;
+
+import com.codeborne.selenide.Condition;
+import com.codeborne.selenide.SelenideElement;
+
+import static com.codeborne.selenide.Selenide.$;
+import static com.codeborne.selenide.Selenide.$$;
+
+public class UsersManagementItem {
+  private final SelenideElement elt;
+
+  public UsersManagementItem(SelenideElement elt) {
+    this.elt = elt;
+  }
+
+  public UsersManagementItem hasTokensCount(Integer count) {
+    this.elt.$(".js-user-tokens").should(Condition.exist).parent().shouldHave(Condition.text(count.toString()));
+    return this;
+  }
+
+  public UsersManagementItem generateToken(String name) {
+    this.openTokenModal();
+    $(".modal #generate-token-form input").should(Condition.exist).sendKeys(name);
+    $(".modal #generate-token-form .js-generate-token").should(Condition.exist).click();
+    $(".modal code").should(Condition.exist);
+    this.getTokenRow(name).should(Condition.exist);
+    this.closeModal();
+    return this;
+  }
+
+  public UsersManagementItem revokeToken(String name) {
+    this.openTokenModal();
+    SelenideElement tokenRow = this.getTokenRow(name).should(Condition.exist);
+    tokenRow.$("button").should(Condition.exist).shouldHave(Condition.text("Revoke")).click();
+    tokenRow.$("button").shouldHave(Condition.text("Sure?")).click();
+    this.getTokenRow(name).shouldNot(Condition.exist);
+    this.closeModal();
+    return this;
+  }
+
+  public UsersManagementItem changePassword(String oldPwd, String newPwd) {
+    this.elt.$("button.dropdown-toggle").should(Condition.exist).click();
+    this.elt.$(".js-user-change-password").shouldBe(Condition.visible).click();
+    this.isModalOpen("Change password");
+    $(".modal #old-user-password").should(Condition.exist).sendKeys(oldPwd);
+    $(".modal #user-password").should(Condition.exist).sendKeys(newPwd);
+    $(".modal #confirm-user-password").should(Condition.exist).sendKeys(newPwd);
+    $(".modal .js-confirm").click();
+    $(".modal").shouldNot(Condition.exist);
+    return this;
+  }
+
+  public UsersManagementItem deactivateUser() {
+    this.elt.$("button.dropdown-toggle").should(Condition.exist).click();
+    this.elt.$(".js-user-deactivate").should(Condition.exist).click();
+    this.isModalOpen("Deactivate User");
+    $(".modal .js-confirm").should(Condition.exist).click();
+    return this;
+  }
+
+  private void openTokenModal() {
+    if (!$(".modal").exists()) {
+      this.elt.$(".js-user-tokens").should(Condition.exist).click();
+    }
+    this.isModalOpen("Tokens");
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Change this instance-reference to a static reference. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AS2209)